build.sbt

```diff
@@ -206,6 +207,7 @@ lazy val scioTest: Project = Project(
       "org.scalatest" %% "scalatest" % scalaTestVersion,
       // DataFlow testing requires junit and hamcrest
       "junit" % "junit" % junitVersion,
+      "com.novocode" % "junit-interface" % junitInterfaceVersion,
```

Comment:
Without it, I believe:
 * no JUnit tests are actually run via SBT
 * there is no feedback from JUnit tests